.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -107,23 +120,36 @@ steps:
           - ".buildkite/auditbeat/scripts/unit-tests.sh"
         notify:
           - github_commit_status:
-              context: "Auditbeat/Extended: MacOS Unit Tests"
+              context: "Auditbeat: MacOS Unit Tests"
```

Comment:
I wonder whether this particular MacOS could be removed, it's already running on GitHub actions:
- https://github.com/elastic/beats/blob/fe132dac9ecaec1efdd4ee3b5887ae63fdfa2832/.github/workflows/macos-auditbeat.yml#L32

https://github.com/elastic/beats/pull/29032 is the original implementation, but it was never reverted. Maybe it makes sense to remove this from buildkite?

Reply:
I believe we need to discuss it within the team, since that potentially affects other pipelines (to maintain similar approaches). I'd suggest to keep it as it is for time being.

Reply:
IMO, I think it makes sense to remove it or take the decisions here rather than migrating it

Reply:
Team decided to keep all the steps in Buildkite